content/tools/data.js

```diff
@@ -129,6 +129,31 @@ const tools = [
     icons: [
       'img/icons/sun.svg',
     ],
+    datasets: [
+      // Q: what do these refer to?
+      // TODO: probably only one or two relates to CDD/HDD...
+      {
+        slug: 'cb0c5f49-59d2-4f8e-8d9c-a047833f6081',
```

Comment:
@mukhtyar what do these cryptic ids correspond to? Something in the API?

Reply:
Yes it's an identifier for the Dataset Instance in the API e.g. https://api.cal-adapt.org/api/datasets/cb0c5f49-59d2-4f8e-8d9c-a047833f6081/